apps/src/code-studio/pd/workshop_dashboard/components/survey_results/free_response_section.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const FreeResponseSection = React.createClass({
+  propTypes: {
+    questions: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    responseData: React.PropTypes.object.isRequired
+  },
+
+  renderFacilitatorsAndAnswers(answers) {
+    return Object.keys(answers).map((facilitator_name, i) => {
+      return (
+        <li key={i}>
+          {facilitator_name}
+          <ul>
+            {
+              answers[facilitator_name].map((feedback, j) => {
+                return (
+                  <li key={j}>
+                    {feedback}
+                  </li>
+                );
+              })
+            }
+          </ul>
+        </li>
+      );
+    });
+  },
+
+  renderAnswers(answers) {
+    return answers.map((answer, i) => {
+      return (
+        <li key={i}>
+          {answer}
+        </li>
+      );
+    });
+  },
+
+  render() {
+    return (
+      <div>
+        {
+          this.props.questions.map((question, i) => {
+            return (
+              <div className="well" key={i}>
```

Comment:
Should we use the React-Bootstrap `Well` component here instead of the class name?